client/signup/accordion-form/accordion-form.tsx

```diff
@@ -3,20 +3,22 @@ import { ChangeEvent, useState } from 'react';
 import AccordionFormSection from './accordion-form-section';
 import {
 	AccordionSectionProps,
-	sectionGeneratorReturnType,
+	SectionGeneratorReturnType,
```

Comment:
Types are normally Title case, but let me know if there is another reason this was  made simple.